openmetadata-service/src/main/java/org/openmetadata/service/OpenMetadataApplication.java

```diff
@@ -248,6 +253,35 @@ public void run(OpenMetadataApplicationConfig catalogConfig, Environment environ
 
     // Register Auth Handlers
     registerAuthServlets(catalogConfig, environment);
+
+    // bolt
+    if (catalogConfig.getSlackAppConfig() != null)
+      regsiterBoltSlackApplication(catalogConfig.getSlackAppConfig(), environment);
+  }
+
+  private void regsiterBoltSlackApplication(
+      SlackAppConfiguration appConfig, Environment environment) {
+    App slackapp = null;
+    try {
+      slackapp = SlackApp.boltSlackAppRegistration(appConfig);
+      boltSlackServletsRegistration(slackapp, environment);
+    } catch (Exception e) {
+      LOG.error("Failed to register Bolt Slack app", e);
+    }
+  }
+
+  private void boltSlackServletsRegistration(App slackApp, Environment environment) {
+    SlackAppOAuthController slackAppOAuthController = new SlackAppOAuthController(slackApp);
+    environment
+        .servlets()
+        .addServlet("SlackOAuthAppServlet", slackAppOAuthController)
+        .addMapping("/api/slack/install", "/api/slack/callback");
```

Comment:
can we mark these /api/slack/install and other URLs as a final constants and reused else where

Reply:
Noted.